copy_old_es_database.py

```diff
@@ -52,12 +141,37 @@ def main():
     logging.basicConfig(format='%(asctime)-15s %(levelname)s %(message)s',
                         level=logging.INFO)
 
-    # Construct the auth from the old AWS credentials
+    # Construct the auths from the new and old AWS credentials
     old_aws_credentials = boto3.Session(profile_name=OLD_AWS_PROFILE_NAME).get_credentials()
     old_awsauth = AWS4Auth(old_aws_credentials.access_key,
                            old_aws_credentials.secret_key,
                            ES_REGION, 'es',
                            session_token=old_aws_credentials.token)
+    new_aws_credentials = boto3.Session().get_credentials()
+    new_awsauth = AWS4Auth(new_aws_credentials.access_key,
+                           new_aws_credentials.secret_key,
+                           ES_REGION, 'es',
+                           session_token=new_aws_credentials.token)
+
+    # Check if the index exists and create it if necessary
+    index_only_url = NEW_ES_URL
+    response = requests.head(index_only_url,
+                             auth=new_awsauth,
+                             timeout=300)
+    if response.status_code != 200:
+        logging.info("The index 'dmarc_aggregate_reports' does not exist.  Creating it.")
+        try:
+            response = requests.put(index_only_url,
+                                    auth=new_awsauth,
+                                    json=INDEX_PAYLOAD,
+                                    headers={'Content-Type': 'application/json'},
+                                    timeout=300)
+            # Raises an exception if we didn't get back a
+            # 200 code
+            response.raise_for_status()
+        except requests.exceptions.RequestException:
+            logging.exception("Unable to create the index 'report.")
```

Comment:
Shouldn't this be:
```
logging.exception("Unable to create the index 'dmarc_aggregate_reports'.")
```
Or am I misunderstanding something?

Reply:
Good eye.  Fixed in 0c4fa15.